configuration-api/main/advancedAuth.js

```diff
@@ -36,16 +36,20 @@ const typeDefs = gql`
     language: String!
     lastTenantSelected: String
   }
+
+  type ResourceEndpoint {
+    url: String!
+    resourceID: String!
```

Comment:
if the resource endpoint is linked to resource type via a relation (parent->child), I doubt you need this.

Reply:
> if the resource endpoint is linked to resource type via a relation (parent->child), I doubt you need this.

Is the declaration of the schema, so is needed for defining the parameter typeOf inside the ResourceType and also the type of data returned by these functions:

-    deleteThisEndpoint(resourceID: [String]!): [ResourceEndpoint]
-     addEndpoint(resourceID: String!, url: String!): [ResourceEndpoint]
-     updateThisEndpoint(resourceID: String!, url: String!): [ResourceEndpoint]
-     getEndpoints(resourceID: String!): [ResourceEndpoint]

If the data returned by these functions or the data inside: endpoint: ResourceEndpoint! is not compatible with the schema graphql will return null